docs/migration-progress.md

```diff
@@ -0,0 +1,40 @@
+# Migration progress
+
+UCX tracks migration progress of _business_ resources: workspace resources that contribute to business value.
+(The term "business resource" comes from the UCX team and is **not** Databricks terminology.) We identified the
+following business resources:
+
+| Workspace resource         | Motivation                                                                                                                  |
+|----------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| Dashboard                  | Dashboards visualize data models supporting business processes                                                              |
+| Job                        | Jobs create data models supporting business process - not exclusively data models used by dashboards.                       |
+| Delta Live Table pipelines | Delta Live Table pipelines create data models supporting business process - not exclusively data models used by dashboards. |
+
+## Usage
+
+Use the migration progress through the [migration progress dashboard](../README.md#dashboards).
```

Comment:
It only becomes available once the preconditions have been met:
a) The `assessment` workflow has completed;
b) A metastore has been attached.

In addition it's currently experimental and needs to be run manually. In a future release it will be scheduled to run daily.